runtime/appruntime/api/handler_test.go

```diff
@@ -86,10 +87,11 @@ func TestDesc_EndToEnd(t *testing.T) {
 		Runtime: &config.Runtime{},
 	}
 	logger := zerolog.New(os.Stdout)
+	customMetrics := custommetrics.NewManager("", "", logger)
```

Comment:
Since we don't pass in a cloud environment, the custom metrics manager won't be initialized. This makes a test fail.

Do we use `local` for testing? Or should the functions that emit metrics check whether the manager is `nil`?

Reply:
We've decided to have a metrics exporter for tests, so that we can test that metrics get emitted when an endpoint is called, for example.